server/mongo.js

```diff
@@ -1,8 +1,8 @@
 //TODO: refactor here
 const mongoose = require('mongoose');
-let testModel = mongoose.model('users', (req,res)=>{
-    name:String
-})
+var schema = new mongoose.Schema({ name: 'string'});
+var User = mongoose.model('user', schema);
+
```

Comment:
Lets do this as a Object or class and export it into index.js
Create the **connect** method, which needs to return a Promise

note: you may use **mongoose.connect('mongodb://...').exec()** to return Promise

Reply:
Works without exec() Done